### PR TITLE
implemented post and get messages backend

### PR DESCRIPTION
**Pull request by psandhu101**
pls review.

### CODE REVIEW

File: backend/app.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import 'reflect-metadata';
-import { createExpressServer } from 'routing-controllers';
```
Comment (ljabdo) [inline backend/app.ts:2]: What is this import used for sir.

**Comment by ljabdo** [comment]
Hey I think you need to run npm format.

**Review by ljabdo — COMMENTED** [review]
lgtm - fix format

File: backend/app.ts (hunk 1 above)
Comment (Darrick-Oliver) [reply backend/app.ts:2]: Looks fine to me

**Pull request merged**